src/commands/move.ts

```diff
@@ -0,0 +1,96 @@
+import ValClient from "../ValClient";
+import { Command, CommandContext } from "../structures";
+import { MusicController } from "../controllers";
+import { log, reply } from "../utils/general";
+import { TextChannel } from "discord.js";
+
+export default class Move extends Command {
+	constructor(client: ValClient) {
+		super(client, {
+			name: "move",
+			category: "Music",
+			cooldown: 5 * 1000,
+			nOfParams: 2,
+			description:
+				"moves a certain track from its position to a specified position",
```

Comment:
`"Moves tracks <from> <to> in the queue. Supports playing a track next when passed 1 parameter only."`